.trunk/configs/.clang-format

```diff
@@ -3,4 +3,4 @@ IndentWidth: 4
 ColumnLimit: 130
 PointerAlignment: Right
 BreakBeforeBraces: Linux
-AllowShortFunctionsOnASingleLine: Inline
+AllowShortFunctionsOnASingleLine: Inline
```

Comment:
Please revert the changes to the .trunk and .vscode files - these changes are unrelated to this PR